### ISSUE TITLE
Pre-work Checklist: Developer: Drake Nguyen

### ISSUE THREAD

**Issue by drakenguyen4000** [hackforla/website#4174]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] #4206
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues in the [prioritized backlog](https://github.com/hackforla/website/projects/7#column-7198257) only with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by drakenguyen4000** [comment]
Progress: I just assigned myself this issue yesterday.  
Availability: Saturday and Sunday.
ETA: Monday 3/20/23

**Comment by drakenguyen4000** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

9 hours

**Comment by jdingeman** [comment]
Hi @drakenguyen4000, I know you're active so I am removing the 2 weeks inactive label. Be sure to provide us with an update on your medium issue! 😄

**Comment by jdingeman** [comment]
Reaching out via Slack for an update.

**Comment by drakenguyen4000** [comment]
Update:
- I'm still waiting on a large issue to work on.  
- I will start reading about Figma and complete by 4/28.  
- Looking to review a small or large issue by 4/28.

**Comment by roslynwythe** [comment]
Hi @drakenguyen4000 congratulations on completing your `Complexity: Large` issue and joining merge team.  Please leave an update at least every two weeks in this issue, unless you are working on other issues in that time period, to avoid having a '2 weeks inactive' label applied to the issue.   After posting an update, please don't forget to remove the '2 weeks inactive' label.  Thank you.

**Comment by HackforLABot** [comment]
## Activity Log: drakenguyen4000
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- drakenguyen4000 opened: https://github.com/hackforla/website/issues/4174 at 03/13/2023, 08:43 PM PDT
- drakenguyen4000 assigned: https://github.com/hackforla/website/issues/4174 at 03/13/2023, 08:45 PM PDT
- drakenguyen4000 assigned: https://github.com/hackforla/website/issues/3877 at 03/16/2023, 08:57 PM PDT
- drakenguyen4000 unassigned: https://github.com/hackforla/website/issues/3877#issuecomment-1473089388 at 03/16/2023, 09:04 PM PDT
- drakenguyen4000 assigned: https://github.com/hackforla/website/issues/3904 at 03/16/2023, 09:06 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/4174#issuecomment-1474673313 at 03/17/2023, 08:52 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/4174#issuecomment-1474679450 at 03/17/2023, 09:06 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/3904#issuecomment-1474684638 at 03/17/2023, 09:10 PM PDT
- drakenguyen4000 opened: https://github.com/hackforla/website/pull/4214 at 03/19/2023, 03:22 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4214#issuecomment-1477222751 at 03/20/2023, 08:09 PM PDT
- drakenguyen4000 PR was merged: https://github.com/hackforla/website/pull/4214#event-8819797967 at 03/22/2023, 11:00 AM PDT
- drakenguyen4000 issue closed by PR 4214: https://github.com/hackforla/website/issues/3904#event-8819798308 at 03/22/2023, 11:00 AM PDT
- drakenguyen4000 assigned: https://github.com/hackforla/website/issues/4268 at 03/23/2023, 08:58 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/4268#issuecomment-1483995312 at 03/25/2023, 09:54 PM PDT
- drakenguyen4000 opened: https://github.com/hackforla/website/pull/4290 at 03/25/2023, 09:55 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4295#issuecomment-1484405732 at 03/26/2023, 07:54 PM PDT
- drakenguyen4000 submitted review: https://github.com/hackforla/website/pull/4295#pullrequestreview-1358315229 at 03/26/2023, 09:42 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4295#issuecomment-1484484753 at 03/26/2023, 09:51 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4290#issuecomment-1486167198 at 03/27/2023, 08:57 PM PDT
- drakenguyen4000 PR was merged: https://github.com/hackforla/website/pull/4290#event-8870290067 at 03/28/2023, 03:51 PM PDT
- drakenguyen4000 issue closed by PR 4290: https://github.com/hackforla/website/issues/4268#event-8870290202 at 03/28/2023, 03:51 PM PDT
- drakenguyen4000 assigned: https://github.com/hackforla/website/issues/4159 at 03/28/2023, 07:21 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/4159#issuecomment-1487922495 at 03/28/2023, 09:15 PM PDT
- drakenguyen4000 opened: https://github.com/hackforla/website/pull/4353 at 03/30/2023, 10:05 PM PDT
- drakenguyen4000 PR was merged: https://github.com/hackforla/website/pull/4353#event-8907891785 at 04/02/2023, 07:46 PM PDT
- drakenguyen4000 issue closed by PR 4353: https://github.com/hackforla/website/issues/4159#event-8907891850 at 04/02/2023, 07:46 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4374#issuecomment-1493664892 at 04/02/2023, 10:03 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4384#issuecomment-1493670671 at 04/02/2023, 10:08 PM PDT
- drakenguyen4000 submitted review: https://github.com/hackforla/website/pull/4384#pullrequestreview-1370117555 at 04/03/2023, 08:07 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4384#issuecomment-1495281714 at 04/03/2023, 08:09 PM PDT
- drakenguyen4000 assigned: https://github.com/hackforla/website/issues/4351 at 04/03/2023, 08:39 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/4351#issuecomment-1495300299 at 04/03/2023, 08:40 PM PDT
- drakenguyen4000 submitted review: https://github.com/hackforla/website/pull/4374#pullrequestreview-1370167366 at 04/03/2023, 09:18 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4374#issuecomment-1495326944 at 04/03/2023, 09:23 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/4351#issuecomment-1500785418 at 04/07/2023, 09:16 PM PDT
- drakenguyen4000 opened: https://github.com/hackforla/website/pull/4444 at 04/08/2023, 08:55 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4437#issuecomment-1501375828 at 04/09/2023, 09:24 PM PDT
- drakenguyen4000 submitted review: https://github.com/hackforla/website/pull/4437#pullrequestreview-1377091297 at 04/09/2023, 10:17 PM PDT
- drakenguyen4000 PR was merged: https://github.com/hackforla/website/pull/4444#event-8979550812 at 04/11/2023, 06:12 PM PDT
- drakenguyen4000 issue closed by PR 4444: https://github.com/hackforla/website/issues/4351#event-8979551029 at 04/11/2023, 06:12 PM PDT
- drakenguyen4000 submitted review: https://github.com/hackforla/website/pull/4437#pullrequestreview-1382485781 at 04/12/2023, 07:31 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4518#issuecomment-1509497986 at 04/14/2023, 09:27 PM PDT
- drakenguyen4000 submitted review: https://github.com/hackforla/website/pull/4518#pullrequestreview-1386304878 at 04/14/2023, 09:28 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4516#issuecomment-1509509609 at 04/14/2023, 09:57 PM PDT
- drakenguyen4000 submitted review: https://github.com/hackforla/website/pull/4516#pullrequestreview-1386311842 at 04/14/2023, 10:01 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/4174#issuecomment-1522648344 at 04/25/2023, 06:54 PM PDT
- drakenguyen4000 assigned: https://github.com/hackforla/website/issues/4568 at 04/29/2023, 03:25 PM PDT
- drakenguyen4000 assigned: https://github.com/hackforla/website/issues/4527 at 04/30/2023, 03:00 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/4527#issuecomment-1529149476 at 04/30/2023, 03:02 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/4568#issuecomment-1529275002 at 04/30/2023, 07:38 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4586#issuecomment-1530788898 at 05/01/2023, 07:38 PM PDT
- drakenguyen4000 submitted review: https://github.com/hackforla/website/pull/4586#pullrequestreview-1408350292 at 05/01/2023, 08:39 PM PDT
- drakenguyen4000 submitted review: https://github.com/hackforla/website/pull/4586#pullrequestreview-1410109090 at 05/02/2023, 07:32 PM PDT
- drakenguyen4000 opened: https://github.com/hackforla/website/issues/4594 at 05/02/2023, 08:19 PM PDT
- drakenguyen4000 assigned: https://github.com/hackforla/website/issues/4594 at 05/02/2023, 08:19 PM PDT
- drakenguyen4000 closed as completed: https://github.com/hackforla/website/issues/4568#event-9151268034 at 05/02/2023, 08:19 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/4594#issuecomment-1533855474 at 05/03/2023, 03:55 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/4527#issuecomment-1536981659 at 05/05/2023, 07:00 PM PDT
- drakenguyen4000 unassigned: https://github.com/hackforla/website/issues/4594 at 05/07/2023, 09:42 AM PDT
- drakenguyen4000 issue closed as completed: https://github.com/hackforla/website/issues/4568#event-9184606701 at 05/07/2023, 09:49 AM PDT
- drakenguyen4000 opened: https://github.com/hackforla/website/pull/4613 at 05/08/2023, 08:20 PM PDT
- drakenguyen4000 PR was merged: https://github.com/hackforla/website/pull/4613#event-9241835232 at 05/14/2023, 09:05 PM PDT
- drakenguyen4000 issue closed by PR 4613: https://github.com/hackforla/website/issues/4527#event-9241835366 at 05/14/2023, 09:05 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4751#issuecomment-1575331710 at 06/03/2023, 07:15 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/4751#event-9423432696 at 06/03/2023, 07:15 PM PDT
- drakenguyen4000 closed by PR 4751: https://github.com/hackforla/website/issues/4669#event-9423432713 at 06/03/2023, 07:15 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4757#issuecomment-1575336013 at 06/03/2023, 07:28 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/4757#event-9423457217 at 06/03/2023, 07:29 PM PDT
- drakenguyen4000 closed by PR 4757: https://github.com/hackforla/website/issues/4670#event-9423457327 at 06/03/2023, 07:29 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4772#issuecomment-1585969291 at 06/10/2023, 06:49 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/4772#event-9492138549 at 06/10/2023, 06:49 PM PDT
- drakenguyen4000 closed by PR 4772: https://github.com/hackforla/website/issues/4416#event-9492138629 at 06/10/2023, 06:49 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4796#issuecomment-1595937554 at 06/17/2023, 08:18 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/4796#event-9560042415 at 06/17/2023, 08:18 PM PDT
- drakenguyen4000 closed by PR 4796: https://github.com/hackforla/website/issues/3975#event-9560042438 at 06/17/2023, 08:18 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4817#issuecomment-1595939867 at 06/17/2023, 08:37 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/4817#event-9560059125 at 06/17/2023, 08:37 PM PDT
- drakenguyen4000 closed by PR 4817: https://github.com/hackforla/website/issues/4514#event-9560059144 at 06/17/2023, 08:37 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4875#issuecomment-1605858410 at 06/24/2023, 09:00 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/4875#event-9628582908 at 06/24/2023, 09:01 PM PDT
- drakenguyen4000 closed by PR 4875: https://github.com/hackforla/website/issues/4551#event-9628582936 at 06/24/2023, 09:01 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4866#issuecomment-1605861299 at 06/24/2023, 09:16 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/4866#event-9628600636 at 06/24/2023, 09:17 PM PDT
- drakenguyen4000 closed by PR 4866: https://github.com/hackforla/website/issues/4787#event-9628600660 at 06/24/2023, 09:17 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4897#issuecomment-1616332576 at 07/01/2023, 08:36 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/4897#event-9698572243 at 07/01/2023, 08:36 PM PDT
- drakenguyen4000 closed by PR 4897: https://github.com/hackforla/website/issues/4859#event-9698572317 at 07/01/2023, 08:36 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4890#issuecomment-1616336586 at 07/01/2023, 08:39 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/4890#event-9698583189 at 07/01/2023, 08:40 PM PDT
- drakenguyen4000 closed by PR 4890: https://github.com/hackforla/website/issues/4883#event-9698583250 at 07/01/2023, 08:40 PM PDT
- drakenguyen4000 closed as completed: https://github.com/hackforla/website/issues/4174#event-9761058881 at 07/07/2023, 07:38 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/4926#issuecomment-1636967930 at 07/15/2023, 08:06 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/4926#event-9830276834 at 07/15/2023, 08:06 PM PDT
- drakenguyen4000 closed by PR 4926: https://github.com/hackforla/website/issues/4507#event-9830276847 at 07/15/2023, 08:06 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5044#event-9892093712 at 07/22/2023, 09:23 PM PDT
- drakenguyen4000 closed by PR 5044: https://github.com/hackforla/website/issues/4920#event-9892093727 at 07/22/2023, 09:23 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/5105#issuecomment-1657038326 at 07/29/2023, 09:18 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5105#event-9956076006 at 07/29/2023, 09:18 PM PDT
- drakenguyen4000 closed by PR 5105: https://github.com/hackforla/website/issues/5032#event-9956076026 at 07/29/2023, 09:18 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/5249#issuecomment-1685166438 at 08/19/2023, 08:34 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5249#event-10137383002 at 08/19/2023, 08:34 PM PDT
- drakenguyen4000 closed by PR 5249: https://github.com/hackforla/website/issues/5174#event-10137383022 at 08/19/2023, 08:34 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5251#event-10201145885 at 08/26/2023, 10:09 PM PDT
- drakenguyen4000 closed by PR 5251: https://github.com/hackforla/website/issues/5042#event-10201145903 at 08/26/2023, 10:09 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5454#event-10325447782 at 09/09/2023, 09:07 PM PDT
- drakenguyen4000 closed by PR 5454: https://github.com/hackforla/website/issues/5411#event-10325447836 at 09/09/2023, 09:07 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5457#event-10390589214 at 09/16/2023, 06:30 PM PDT
- drakenguyen4000 closed by PR 5457: https://github.com/hackforla/website/issues/5368#event-10390589236 at 09/16/2023, 06:30 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5523#event-10390591195 at 09/16/2023, 06:32 PM PDT
- drakenguyen4000 closed by PR 5523: https://github.com/hackforla/website/issues/5459#event-10390591225 at 09/16/2023, 06:32 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5690#event-10656083498 at 10/14/2023, 04:01 PM PDT
- drakenguyen4000 closed by PR 5690: https://github.com/hackforla/website/issues/5622#event-10656083516 at 10/14/2023, 04:01 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5688#event-10656086483 at 10/14/2023, 04:04 PM PDT
- drakenguyen4000 closed by PR 5688: https://github.com/hackforla/website/issues/5633#event-10656086513 at 10/14/2023, 04:04 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5683#event-10656088559 at 10/14/2023, 04:06 PM PDT
- drakenguyen4000 closed by PR 5683: https://github.com/hackforla/website/issues/5336#event-10656088589 at 10/14/2023, 04:06 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5682#event-10656092870 at 10/14/2023, 04:11 PM PDT
- drakenguyen4000 closed by PR 5682: https://github.com/hackforla/website/issues/5617#event-10656092881 at 10/14/2023, 04:11 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5583#event-10656119716 at 10/14/2023, 04:21 PM PDT
- drakenguyen4000 closed by PR 5583: https://github.com/hackforla/website/issues/5506#event-10656119740 at 10/14/2023, 04:21 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/5916#issuecomment-1817768732 at 11/18/2023, 11:04 PM PST
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5916#event-11003174317 at 11/18/2023, 11:04 PM PST
- drakenguyen4000 closed by PR 5916: https://github.com/hackforla/website/issues/5791#event-11003174341 at 11/18/2023, 11:04 PM PST
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/5887#issuecomment-1817770573 at 11/18/2023, 11:13 PM PST
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5887#event-11003188346 at 11/18/2023, 11:13 PM PST
- drakenguyen4000 closed by PR 5887: https://github.com/hackforla/website/issues/5808#event-11003188367 at 11/18/2023, 11:13 PM PST
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/5893#event-11003195010 at 11/18/2023, 11:18 PM PST
- drakenguyen4000 closed by PR 5893: https://github.com/hackforla/website/issues/5803#event-11003195035 at 11/18/2023, 11:18 PM PST
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6014#event-11331869497 at 12/23/2023, 08:01 PM PST
- drakenguyen4000 closed by PR 6014: https://github.com/hackforla/website/issues/5849#event-11331869513 at 12/23/2023, 08:01 PM PST
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/6360#issuecomment-1962925353 at 02/25/2024, 04:38 AM PST
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6360#event-11912915441 at 02/25/2024, 04:39 AM PST
- drakenguyen4000 closed by PR 6360: https://github.com/hackforla/website/issues/5711#event-11912915476 at 02/25/2024, 04:39 AM PST
- drakenguyen4000 commented on: https://github.com/hackforla/website/pull/6462#issuecomment-2002319754 at 03/16/2024, 10:42 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6462#event-12141991474 at 03/16/2024, 10:43 PM PDT
- drakenguyen4000 closed by PR 6462: https://github.com/hackforla/website/issues/6303#event-12141991498 at 03/16/2024, 10:43 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6411#event-12302220558 at 03/30/2024, 08:05 PM PDT
- drakenguyen4000 closed by PR 6411: https://github.com/hackforla/website/issues/6299#event-12302220578 at 03/30/2024, 08:05 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6467#event-12302223441 at 03/30/2024, 08:08 PM PDT
- drakenguyen4000 closed by PR 6467: https://github.com/hackforla/website/issues/4648#event-12302223454 at 03/30/2024, 08:08 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6508#event-12302225373 at 03/30/2024, 08:10 PM PDT
- drakenguyen4000 closed by PR 6508: https://github.com/hackforla/website/issues/6484#event-12302225388 at 03/30/2024, 08:10 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6521#event-12302225921 at 03/30/2024, 08:11 PM PDT
- drakenguyen4000 closed by PR 6521: https://github.com/hackforla/website/issues/6510#event-12302225950 at 03/30/2024, 08:11 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6536#event-12302227370 at 03/30/2024, 08:12 PM PDT
- drakenguyen4000 closed by PR 6536: https://github.com/hackforla/website/issues/6479#event-12302227381 at 03/30/2024, 08:12 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6543#event-12302229935 at 03/30/2024, 08:15 PM PDT
- drakenguyen4000 closed by PR 6543: https://github.com/hackforla/website/issues/5473#event-12302229943 at 03/30/2024, 08:15 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6631#event-12459470358 at 04/13/2024, 09:53 PM PDT
- drakenguyen4000 closed by PR 6631: https://github.com/hackforla/website/issues/6177#event-12459470426 at 04/13/2024, 09:53 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6630#event-12459475704 at 04/13/2024, 09:55 PM PDT
- drakenguyen4000 closed by PR 6630: https://github.com/hackforla/website/issues/6178#event-12459475775 at 04/13/2024, 09:55 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/5885#issuecomment-2060257248 at 04/16/2024, 07:59 PM PDT
- drakenguyen4000 commented on: https://github.com/hackforla/website/issues/6493#issuecomment-2060263672 at 04/16/2024, 08:06 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6890#event-12934975962 at 05/25/2024, 09:29 PM PDT
- drakenguyen4000 closed by PR 6890: https://github.com/hackforla/website/issues/6847#event-12934976007 at 05/25/2024, 09:29 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6874#event-12935000057 at 05/25/2024, 09:37 PM PDT
- drakenguyen4000 closed by PR 6874: https://github.com/hackforla/website/issues/6720#event-12935000100 at 05/25/2024, 09:37 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/6991#event-13173639917 at 06/15/2024, 11:13 PM PDT
- drakenguyen4000 closed by PR 6991: https://github.com/hackforla/website/issues/6737#event-13173639988 at 06/15/2024, 11:13 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/7038#event-13253123626 at 06/22/2024, 09:17 PM PDT
- drakenguyen4000 closed by PR 7038: https://github.com/hackforla/website/issues/6932#event-13253123646 at 06/22/2024, 09:17 PM PDT
- drakenguyen4000 merged: https://github.com/hackforla/website/pull/7040#event-13341168933 at 06/29/2024, 08:16 PM PDT
- drakenguyen4000 closed by PR 7040: https://github.com/hackforla/website/issues/6986#event-13341168949 at 06/29/2024, 08:16 PM PDT
